test/e2e_node/log_path_test.go

```diff
@@ -69,6 +70,7 @@ var _ = framework.KubeDescribe("ContainerLogPath [NodeConformance]", func() {
 
 				logDirVolumeName := "log-dir-vol"
 				logDir := kubelet.ContainerLogsDir
+				logCRIDir := "/var/log/pods"
```

Comment:
Create new test case, with helper function for both the existing and new CRI log path.

Reply:
You mean just like the table-driven test, which is organized as following, 
```
tcs := []struct {
       in string
       exp string
} {
    {in:"in1", exp:"out1"},
    {in:"in2", exp:"out2"},
}

for i, tc := range tcs {
       out := testFunc(tc.in)
       if exp != out {
              t.Errorf(...)
       }
}

```

Reply:
No, create individual test cases.

Reply:
Sorry, I'm still confused with "individual test cases".

Does "individual test cases" mean like this ?

```
Context("printed log to stdout", func() {
    It("should print log to correct log path", func() {
         // run with common helper function
    }
    It("should print log to correct cri log path", func() {
         // run with common helper function
    }
}
```

Reply:
ok，thank you，I will modify like this